### PR TITLE
Update prevent-end-user-interaction-microsoft-defender-antivirus.md

### LINKED ISSUE #12490: Suggestion to also add local command using Powershell to adjust UILockdown
I am working with a customer having UILockdown issue, where they checked GPO and found no policy on the UILockdown. However, when we checked Get-MpPreference we can see local device setting is set to true. 

I think it will be super helpful to others having similar issue where UILockdown is for some reason enabled locally from the device and they need to change them. We can use the following command:

**Set-MpPreference -UILockdown 0**

And use **Get-MpPreference t**o check the value of UILockdown
image

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9870c42b-eede-b869-3d4d-cd2c53be82ed
* Version Independent ID: 8ae67d57-bb69-1e7c-92df-2960e2a2f152
* Content: [Hide the Microsoft Defender Antivirus interface](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
Added UI lockdown cmdlet

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12490

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -98,6 +98,15 @@ You can prevent users from pausing scans, which can be helpful to ensure schedul
 > - [Configure Defender for Endpoint on Android features](android-configure.md)
 > - [Configure Microsoft Defender for Endpoint on iOS features](ios-configure-features.md)
 
+
+## UI Lockdown
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md]:
```suggestion
## UI Lockdown mode
```

File: microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md [hunk 2]
```diff
@@ -98,6 +98,15 @@ You can prevent users from pausing scans, which can be helpful to ensure schedul
...
+
+Indicates whether to disable UI lockdown mode. If you specify a value of $True, Windows Defender disables UI lockdown mode. If you specify $False or do not specify a value, UI lockdown mode is enabled.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prevent-end-user-interaction-microsoft-defender-antivirus.md]:
```suggestion
Indicates whether to disable UI Lockdown mode. If you specify a value of $True, Windows Defender disables UI Lockdown mode. If you specify $False or do not specify a value, UI Lockdown mode is enabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**